tests/components/kaleidescape/conftest.py

```diff
@@ -17,7 +17,7 @@
 
 
 @pytest.fixture(name="mock_device")
-def fixture_mock_device() -> Generator[None, AsyncMock, None]:
+def fixture_mock_device() -> Generator[AsyncMock]:
```

Comment:
This is returning a class - I don't think it should be `AsyncMock`
```suggestion
def fixture_mock_device() -> Generator[MagicMock]:
```

Reply:
#119040